mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -192,6 +193,13 @@ uploadSingle file a = do
   x <- liftIO $ LBS.readFile file
   void . send $ fencode' putObject a (toBody x) & poServerSideEncryption .~ pure sse
 
+-- | Create a conduit sink for a given S3 Address
+s3Sink :: Address -> Sink BS.ByteString AWS ()
+s3Sink a =
+  do bs <- C.await
+     lift $ maybe (pure ()) sendRequest bs
+     where sendRequest bs = void . send $ fencode' putObject a (toBody bs) & poServerSideEncryption .~ pure sse
```

Comment:
(either way, a good trigger that this needs a property that test large files similar to multipart upload/download)